examples/mobile/HomeApp/index.html

```diff
@@ -34,8 +34,10 @@
 		</header>
 		<div class="ui-content">
 			<div class="ui-content-area">
-				Weather Widget
-			</div>
+				<a class="weatherwidget-io" href="https://forecast7.com/en/37d57126d98/seoul/" data-label_1="Seoul" data-label_2="South Korea" data-theme="original" >Seoul South Korea</a>
+				<script>
+				!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0];if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src='https://weatherwidget.io/js/widget.min.js';fjs.parentNode.insertBefore(js,fjs);}}(document,'script','weatherwidget-io-js');
```

Comment:
If we need this widget, please indent check.

Reply:
done